src/Http/Controllers/Baskets/BasketLineController.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+namespace GetCandy\Api\Http\Controllers\Baskets;
+
+use Illuminate\Support\Facades\Request;
+use GetCandy\Api\Http\Controllers\BaseController;
+use GetCandy\Api\Http\Resources\Baskets\BasketResource;
+use GetCandy\Api\Core\Baskets\Services\BasketLineService;
+use GetCandy\Api\Http\Requests\Baskets\UpdateLineRequest;
+use GetCandy\Api\Core\Baskets\Factories\BasketLineFactory;
+use GetCandy\Api\Http\Requests\Baskets\CreateLinesRequest;
+use GetCandy\Api\Http\Requests\Baskets\DeleteLinesRequest;
+use GetCandy\Api\Http\Requests\Baskets\ChangeQuantityRequest;
+
+
+class BasketLineController extends BaseController
+{
+    /**
+     * @var BasketLineFactory
+     */
+    protected $factory;
+
+    /**
+     * @var BasketLineService $basketLines
+     */
+    protected $basketLines;
+
+    public function __construct(BasketLineFactory $factory)
+    {
+        $this->factory = $factory;
+        $this->basketLines = app('api')->basketLines();
+        $this->basketLines->setIncludes(Request::get('includes') ?? '');
```

Comment:
This seems to cause an error when no includes are passed, if no includes are passed we could just pass an empty array `[]`?

Reply:
I was confused by this, because the includes are a string.